src/com/massivecraft/factions/integration/V19/EngineV19.java

```diff
@@ -0,0 +1,70 @@
+package com.massivecraft.factions.integration.V19;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.entity.AreaEffectCloudApplyEvent;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.projectiles.ProjectileSource;
+
+import com.massivecraft.factions.Factions;
+import com.massivecraft.factions.engine.EngineCombat;
+import com.massivecraft.massivecore.Engine;
+import com.massivecraft.massivecore.MassivePlugin;
+import com.massivecraft.massivecore.util.MUtil;
+
+public class EngineV19 extends Engine
+{
+    // -------------------------------------------- //
+    // INSTANCE & CONSTRUCT
+    // -------------------------------------------- //
+
+    private static EngineV19 i = new EngineV19 ();
+    public static EngineV19 get() { return i; }
+
+    @Override
+    public MassivePlugin getActivePlugin()
+    {
+        return Factions.get();
+    }
+
+    // -------------------------------------------- //
+    // LISTENER
+    // -------------------------------------------- //
```

Comment:
I just noticed that some parts of this class is indented with spaces rather than tabs. (It's probably a copy paste error since you created them from a comment)

Could you change these spaces to tabs? That would be awesome!